packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts

```diff
@@ -52,17 +54,28 @@ export class ExportPanel extends ConnectedLitElement {
     this.config = state.config;
     this.camera = state.camera;
     this.hotspots = state.hotspots;
+    this.gltfUrl = state.gltfUrl;
   }
 
   render() {
     const editedConfig = {...this.config};
     applyCameraEdits(editedConfig, this.camera);
-    // Don't render the data URI. Users should download the edited GLB and put
-    // the right URL here.
-    editedConfig.src = `model.glb`;
-    if (editedConfig.environmentImage) {
-      editedConfig.environmentImage = `environment-image.hdr`;
+
+    // If the last loaded URL is not an object URL, echo it here for convenience.
+    if(this.gltfUrl && !isObjectUrl(this.gltfUrl)) {
+      editedConfig.src = this.gltfUrl;
+    }
+    else {
+      // Uploaded GLB
+      editedConfig.src = `Change this to your GLB URL`;
+    }
+
+    if (editedConfig.environmentImage &&
+      isObjectUrl(editedConfig.environmentImage)) {
+      // Uploaded env image
+      editedConfig.environmentImage = `Change this to your HDR URL`;
```

Comment:
Should we do the same thing here as above, to retain the url if it's not an Object URL?

Reply:
We already do retain it, so this code does that.